campaign-launcher/interfaceV2/src/components/CampaignsTable/index.tsx

```diff
@@ -134,6 +133,61 @@ const CustomPagination: FC<{
   );
 };
 
+const statusTooltipData = [
+  {
+    status: 'Active',
+    color: 'success.main',
+  },
+  {
+    status: 'Awaiting start date',
+    color: 'warning.main',
+  },
+  {
+    status: 'End date is earlier than current date',
```

Comment:
Maybe we should say something like `campaign finished, waiting for payouts`? What we have now is not 100% clear to me